src/couch/src/couch_changes.erl

```diff
@@ -556,9 +552,15 @@ send_changes(Acc, Dir, FirstRound) ->
     end.
 
 
-can_optimize(true, {doc_ids, _Style, DocIds})
-        when length(DocIds) =< ?MAX_DOC_IDS ->
-    {true, fun send_changes_doc_ids/6};
+can_optimize(true, {doc_ids, _Style, DocIds}) ->
+    MaxDocIds = list_to_integer(
+        config:get("changes", "max_doc_ids", "100")
```

Comment:
Thanks for PR.

A few suggestions:

* Instead of `max_doc_ids` maybe make it a bit more descriptive, something like `doc_ids_optimization_threshold`

 * Consider using `config:get_integer` as in:
```
 MaxDocIds = config:get_integer("changes", "max_doc_ids", 100) 
```
 * Make sure to add the section with the default value and a description to the default.ini config file. It's in : `rel/overlay/etc/default.ini`
 
  * We don't have an existing `[changes]` section, wonder if it would be better to add this to `[couchdb]`. I am 50/50 on this, let's see what others think.